### ISSUE TITLE
Towards better scaling tests

### ISSUE THREAD

**Issue by permcody** [idaholab/moose#4984]
Our scaling test suite sucks...

First there are several tests that aren't properly scaled (i.e. they run in a fraction of a second) and secondly we have only one way to scale tests (i.e. make the mesh finer). I think we need to create more realistic scaling tests that are a combination of refining and additional time steps, etc. One way to go about this would be to provide the desired cli-args in the scale_refine parameter.

```
  scale_refine = 'Executioner/num_steps=1000 Mesh/uniform_refine=4 Kernels/coef_diff/coef=1e-3'
```

In other words, we need to make the test designer come up with a good way of scaling individual tests beyond what we have now. Other ideas are welcome.

**Comment by friedmud** [comment]
:+1: i like this idea

**Comment by permcody** [comment]
This is being superseded by using a new "BenchMark" tester capability which we are in the process of rolling out. We plan to turn off frequency scaling, run a specific input file several times and report stats. We aren't going to do benchmarking through scaling like this. Closing...
